etcdserver/auth/auth_test.go

```diff
@@ -357,7 +353,17 @@ func TestEnsure(t *testing.T) {
 	}
 }
 
+type fastPasswordStore struct {
+}
+
+func (_ fastPasswordStore) CheckPassword(user User, password string) bool {
+	return user.Password == password
+}
+
+func (_ fastPasswordStore) HashPassword(password string) (string, error) { return password, nil }
+
 func TestCreateAndUpdateUser(t *testing.T) {
+
```

Comment:
Oh! I get it. It's because I based on Yicheng's branch originally, which had them.